clouddriver-integration/src/test/java/com/netflix/spinnaker/clouddriver/BaseContainerTest.java

```diff
@@ -32,37 +32,36 @@
 import org.testcontainers.containers.GenericContainer;
 import org.testcontainers.containers.Network;
 import org.testcontainers.containers.wait.strategy.Wait;
-import org.testcontainers.junit.jupiter.Testcontainers;
 import org.testcontainers.utility.DockerImageName;
 
-@Testcontainers
 public class BaseContainerTest {
 
-  private static final Logger logger = LoggerFactory.getLogger(BaseContainerTest.class);
+  protected final Logger logger = LoggerFactory.getLogger(BaseContainerTest.class);
```

Comment:
This could still be private/static, yes?

Reply:
Yes. changing it.

Reply:
Done